pkg/nsx/services/common/types.go

```diff
@@ -61,7 +61,9 @@ const (
 	TagValueGroupScope              string = "scope"
 	TagValueGroupSrc                string = "source"
 	TagValueGroupDst                string = "destination"
-	TagValueVersion10               string = "1.0"
+	TagValueMajorVersion            string = "1"
+	TagValueMinorVersion            string = ".0"
+	TagValuePatchVersion            string = ".0"
 	AnnotationVPCNetworkConfig      string = "nsx.vmware.com/vpc_network_config"
```

Comment:
how about:
TagValueMinorVersion  string = "0"
TagValuePatchVersion  string = "0"
define a var
	TagValueVersion10               = [3]string{TagValueMajorVersion, TagValueMinorVersion, TagValuePatchVersion}
when you need version string, 
 strings.Join(TagValueVersion10, ".")

Reply:
fixed